modules/disable-quickstarts-admin-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * web_console/configuring-web-console.adoc
+
+[id="disable-quickstarts-admin-console_{context}"]
+= Disabling quick starts in the web console
+
+You can use the *Administrator* perspective of the web console to disable one or more quick starts.
+
+.Prerequisites
+
+* You have cluster administrator permissions and are logged in to the web console.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Administation* -> *Cluster Settings*.
+. In the *Cluster Settings* page, click the *Configuration* tab.
+. In the *Configuration* page, click the *Console* configuration resource with the description *operator.openshift.io*.
++
+image::cluster-settings-console.png[Image of the Configuration page showing the correct Console configuration resource to select]
+
+. In the *Action* drop-down list, click *Customize*. This takes you to the *Cluster configuration* page.
++
+image::cluster-configuration-general-tab.png[Cluster configuration page open on the General tab]
+
+. In the *General* tab, under *Quick starts*, you can select items in either the *Enabled* or *Disabled* list, and move them from one list to the other by using the arrow buttons.
```

Comment:
```suggestion
. On the *General* tab, in the *Quick starts* section, select items in either the *Enabled* or *Disabled* list, and move them from one list to the other by using the arrow buttons.
```
- Per ISG, use "on" with tabs: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#tab
- Replace "under". Per ISG, for accessibility reasons, we shouldn't refer user interface elements by using sensory words: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements